src/components/LanguageWrapper/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+
+import { preferencesSetLocale } from 'state/actions/preferences';
+import english from 'languages/en';
+import spanish from 'languages/es';
+import { availableLocales } from 'utils/index';
+
+const browserLocale = navigator.language.split(/[-_]/)[0];
+
+const messages = {
```

Comment:
This message object and the languages imports should go to the `utils/` folder. So in the case we need the locales of the app we could call it simply like this: `availableLocales()`.